cmake/3rdParty/Platform/Windows/pix_windows.cmake

```diff
@@ -11,4 +11,4 @@ if(LY_MONOLITHIC_GAME)
 else()
     set(PIX_LIBS ${BASE_PATH}/bin/x64/WinPixEventRuntime.lib)
     set(PIX_RUNTIME_DEPENDENCIES ${BASE_PATH}/bin/x64/WinPixEventRuntime.dll)
-endif()
+endif()
```

Comment:
Why did you get rid of the newline at the end of the file?

Reply:
Autoformatter somewhere (I'll track it down) thanks